sonic-slave-buster/Dockerfile.j2

```diff
@@ -4,6 +4,14 @@ FROM multiarch/debian-debootstrap:armhf-buster
 COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 {%- elif CONFIGURED_ARCH == "arm64" and MULTIARCH_QEMU_ENVIRON == "y" %}
 FROM multiarch/debian-debootstrap:arm64-buster
+{%- elif CONFIGURED_ARCH == "armhf" and CROSS_BUILD_ENVIRON == "y" %}
+FROM multiarch/qemu-user-static:x86_64-arm-5.0.0-2 as qemu
```

Comment:
>5.0.0-2 [](http://example.com/codeflow?start=43&length=7)

Why fixed version?

Reply:
This is reuse (and restructuring) of already used version.

Reply:
Is it possible to follow the latest version?